doc/en/changelog.rst

```diff
@@ -28,6 +28,34 @@ with advance notice in the **Deprecations** section of releases.
 
 .. towncrier release notes start
 
+pytest 5.3.3 (2020-01-16)
+=========================
+
+Bug Fixes
```

Comment:
It's been like that forever, but wouldn't "Bugfixes" be more common?

Reply:
No, "Bug fixes" is more common with 6,063,858 available code results:

https://github.com/search?utf8=%E2%9C%93&q=%22Bug+fixes%22&type=Code

"Bugfix" has 3,876,434:

https://github.com/search?q=Bugfixes&type=Code

Reply:
=> https://github.com/pytest-dev/pytest/pull/6549#discussion_r370273926
As per Google "bugfixes" is the more popular one.